openstack/designate/values.yaml

```diff
@@ -7,6 +7,7 @@ debug: "True"
 
 global_setup: false
 tempest_enabled: false
+nanny_enabled: false
```

Comment:
since this is coming to all region, I think it is fine to leave it to true per default.

Reply:
I just aligned it with tempest and had the idea to have a "safe" default in the helm charts to be overwritten where we need it. in global for example we don't have it in all regions and when its true by default there will be missing configuration.